README.md

```diff
@@ -67,86 +84,58 @@ The table below outlines what version of Nexus Repository the plugin was built a
 | v1.0.2         | >=3.14.0-04              |
 | v1.0.3         | >=3.15.2-01              |
 | v1.0.4         | >=3.18.0-01              |
-
-If a new version of Nexus Repository is released and the plugin needs changes, a new release will be made, and this
-table will be updated to indicate which version of Nexus Repository it will function against. This is done on a time 
-available basis, as this is community supported. If you see a new version of Nexus Repository, go ahead and update the
-plugin and send us a PR after testing it out!
-
+| v1.1.0 In product | >=3.20.0              |
 All released versions can be found [here](https://github.com/sonatype-nexus-community/nexus-repository-r/releases).
 
 ## Installing the plugin
+Starting from Nexus Repository Manager 3.20 `R` format is already included. So there are no need to install it.  But if you want to reinstall plugin with your improvements then next instructions will be useful for you.
 
```

Comment:
But if you want to reinstall **the** plugin ?

Reply:
can we say "then next instructions will be useful for you. **Note: Using an unofficial version of the plugin is not supported by the Sonatype Support team**

Reply:
Trivial suggestion: "following instructions" vs "next instructions".
Also suggest a period at the end of @jlstephens89 note.

Reply:
d5515a87d0a27602c03b170f36e2535cbe3f2ab9